shared/cluster.go

```diff
@@ -35,6 +36,8 @@ type Pod struct {
 	Node      string
 }
 
+type ArgsMap map[string]interface{}
```

Comment:
if you are using this on the aux. should be living there

Reply:
Updated